src/main/generated/resources/data/minecolonies/loot_tables/blocks/blockhutwmysticalsite.json

```diff
@@ -0,0 +1,25 @@
+{
+  "type": "minecraft:block",
+  "pools": [
+    {
```

Comment:
The file name is `wmysticalsite`?

Reply:
Sorry, I corrected it, so much refactor that one escaped me